src/components/views/read/cvForm/Cv.tsx

```diff
@@ -211,58 +211,48 @@ const Template: FC<Props> = ({ nextStep, prevStep }) => {
     (obj) => obj.type === APP_DATA_TYPES.EDUCATION_INFO,
   );
 
-  const educationDataArray: Data[] = [];
-  educationInfoObject.map((card) => {
-    educationDataArray.push(card.data);
-    return null;
-  });
+  const educationDataArray = educationInfoObject.map(
+    (card) => card.data,
+  ) as unknown as EducationInfoObj[];
```

Comment:
Your casting to unknown is an clue to the fact that it is not what you want. I think you meant:
```suggestion
  const educationDataArray = educationInfoObject.map(
    (card) => card.data as EducationInfoObj,
  );
```

Reply:
I tried that actually, and If i set that again to be like that it will get this error:
```
Conversion of type 'List<Data>' to type 'EducationInfoObj' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'List<Data>' is missing the following properties from type 'EducationInfoObj': id, degree, institutionName, major, and 5 more.ts(2352)
const educationInfoObject: Immutable.List<AppData>
```

Reply:
Yes this is why you need to cast it **inside** the map. The issue is that you are getting an Immutable.List which not the same as an Array (`[]`) but they have the same API, so `.map` exists on both. Check my suggestion again.